VantageCloudLakeUseCases/content/ExternalObjectStore/README.md

```diff
@@ -7,7 +7,7 @@ Open Editor to proceed with this use case.
 
 ### Introduction
 
-The following is a summary of how to access different formats of data stored in an object store. You can copy and modify the example queries below to access your own datasets. For simplicity the included datasets are setup to not need credentials, but it is highly recommended that you use credentials to access your own datasets.
+The following is a summary of how to copy data from VantageCloud Lake to an object store. You must provide your own bucket to execute the example queries below. We are using the __sample_sensor__ table to simulate copying data from a table in your vantage system to the external object store. You can copy and modify the example queries below to access your own datasets. For simplicity the included datasets are setup to not need credentials, but it is highly recommended that you use credentials to access your own datasets.
```

Comment:
I would simply this as:

This section shows how to copy data from VantageCloud Lake to an object store. To execute the example queries you must provide your own bucket. The **sample_sensor** table simulates copying data from a table in your Vantage system to the external object store. You can modify these examples to access to your own datasets. Although these example datasets do not require login credentials, you should use login credentials to access your own datasets.

Reply:
@KV250025: 

"... from a table in your vantage system to the external object store."

Is this for **any** Vantage system (in which case, simply capitalize "V"antage, or for VantageClould Lake only (in which case, write out the full product name).

Reply:
It is for any Vantage system. I used a upper case "V"antage in the changes https://github.com/Teradata/product-help/pull/58/files